rules/aip0231/response_resource_field.go

```diff
@@ -31,7 +31,8 @@ var resourceField = &lint.MessageRule{
 		// Note: Use m.GetName()[8 : len(m.GetName())-9] to retrieve the resource
 		// name from the the batch get response, for example:
 		// "BatchGetBooksResponse" -> "Books"
-		resourceMsgName := pluralize.NewClient().Singular(m.GetName()[8 : len(m.GetName())-9])
+		plural := m.GetName()[8 : len(m.GetName())-8]
```

Comment:
The comment is helpful, but why not use `strings.TrimPrefix(n, "BatchGet")` and `strings.TrimSuffix(n, "Response")`? It would end up being more than one line, but it would be more readable and we wouldn't need magic numbers or such a detailed comment. I think the magic number was part of the cause of this bug too anyways.

Reply:
Fair point. Done.